app/controllers/jobs-controller.js

```diff
@@ -34,6 +39,9 @@ router.post('/new', (req, res, next) => {
   const jobData = Object.assign({}, req.body, {
     accountId,
     status_sort_index: progression.getById(req.body.status).order,
+    deadlineDate: parseDateOrReturnNull(req.body.deadlineDate),
+    applicationDate: parseDateOrReturnNull(req.body.applicationDate),
+    interviewDate: parseDateOrReturnNull(req.body.interviewDate),
```

Comment:
Intend to only store dates that have been entered but if i'm adding a job and i'm at the application stage i may well input what the deadline date was as well as the application date and i'm not sure we should restrict that.